modules/refined/src/test/scala/doobie/refined/RefinedSuite.scala

```diff
@@ -54,15 +52,15 @@ class RefinedSuite extends munit.FunSuite {
   }
 
   test("Query should return a refined type when conversion is possible") {
-    sql"select 123".query[PositiveInt].unique.transact(xa).void.unsafeRunSync()
+    sql"select 123".query[PositiveInt].unique.transact(xa).void.assert
```

Comment:
Not sure, why the original test doesn't contain any value checks besides that there's no failure.
I think adding this check shouldn't hurt, should it?
```suggestion
    sql"select 123".query[PositiveInt].unique.transact(xa)
      .assertEquals(PositiveInt(123))
```